test/framework/README.md

```diff
@@ -0,0 +1,38 @@
+# CAPI e2e testing framework
+
+This framework aims to define common end-to-end patterns that can be reused across Cluster API providers.
+
+## Usage
+
+Use this framework as you see fit. If there are pieces that don't work for you, do not use them. If there are pieces
+that almost work for you, customize them. If this does not fit your e2e testing please file an issue and we can discuss
+your use case and find a nice solution for everyone.
+
+### Contents
+
+This framework contains
+
+* An interface to a management cluster
+    * An struct that implements the management cluster interface using `kind` as the backend.
+* A series of behavioral tests
+* Clean up tests.
```

Comment:
Tests for clean up or cleaning up test resources?

Reply:
I've removed this as it's covered in the interface & struct above that i've added links to